dev/com.ibm.ws.javaee.ddmodel.wsbnd/src/com/ibm/ws/javaee/ddmodel/wsbnd/HttpPublishing.java

```diff
@@ -1,7 +1,7 @@
 /*******************************************************************************
- * Copyright (c) 2013 IBM Corporation and others.
+ * Copyright (c) 2013,2020 IBM Corporation and others.
  * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
+ * are made available under the terms of the Eclipse License v1.0
```

Comment:
There are a couple files that changed the copyright header to read 'Eclipse License' rather than 'Eclipse Public License'

Reply:
Fixed.